relayer/cmd/generate_beacon_data.go

```diff
@@ -345,26 +354,31 @@ func generateBeaconTestFixture(cmd *cobra.Command, _ []string) error {
 
 		log.WithFields(log.Fields{
 			"location": pathToBeaconBenchmarkData,
-			"filename": filename,
 		}).Info("writing result file")
 
-		err = writeBenchmarkDataFile(fmt.Sprintf("%s/%s", pathToBeaconBenchmarkData, filename), rendered)
+		err = writeBenchmarkDataFile(fmt.Sprintf("%s", pathToBeaconBenchmarkData), rendered)
 		if err != nil {
 			return err
 		}
 
 		// writing inbound queue fixtures
+		testCase, _ := cmd.Flags().GetString("test_case")
+		if err != nil {
+			return err
+		}
+		pathToInboundBenchmarkDataTemplate := fmt.Sprintf("parachain/templates/%s.mustache", testCase)
+		pathToInboundQueueFixtureData := fmt.Sprintf("parachain/pallets/inbound-queue/fixtures/src/%s.rs", testCase)
```

Comment:
Can use a constant for these paths too.

Reply:
https://github.com/Snowfork/snowbridge/pull/1125/commits/7b1948a3f7e1a585695b7e792db66d292d389587